crypto/ed25519/ed25519.go

```diff
@@ -7,10 +7,15 @@ import (
 
 	"github.com/nknorg/nkn/crypto/ed25519/vrf"
 	"github.com/nknorg/nkn/crypto/util"
+	"github.com/nknorg/vxeddsa/extra25519"
 
 	"golang.org/x/crypto/ed25519"
 )
 
+const PublicKeySize = ed25519.PublicKeySize
```

Comment:
I am afraid that make confuse `ed25519.PublicKeySize` between "golang.org/x/crypto/ed25519" and "gitHub.com/nknorg/nkn/crypto/ed25519"

Reply:
We should be fine as long as we don't confuse the compiler :)

PS: I think this will be removed soon when we drop p256 support